data/mods/Xedra_Evolved/player/professions.json

```
@@ -29,6 +29,46 @@
       { "item": "5x50_50_mag", "ammo-item": "5x50dart", "charges": 50 }
     ]
   },
+  {
+    "type": "profession",
+    "id": "fbi_investigator",
+    "name": "FBI Paranormal Investigator",
+    "description": "Before the Cataclysm, you used to work for Paranormal Investigation Department of FBI.  You got used to jokes at your expense, as well as being constantly underfunded and disregarded by your superiors.  You were returning from yet another case with nothing but a bag of suspicious candies to show, when all hell broke loose.  Oh well, at least now you can be certain that everyone still alive will belive you that the truth is out there.",
```

```suggestion
    "description": "Before the Cataclysm, you used to work for the Paranormal Investigation Department of the FBI.  You got used to jokes at your expense, as well as being constantly underfunded and disregarded by your superiors.  You were returning from yet another case with nothing but a bag of suspicious candies to show, when all hell broke loose.  Oh well, at least now you can be certain that everyone still alive will believe you that the truth is out there.",
```
(FBI Paranormal Investigation Department works too)

---

Oh derp, articles are hard
Thanks for catching the typo too